label.rb

```diff
@@ -0,0 +1,17 @@
+class Label
+  attr_accessor :title, :color, :items, :id
+
+  def initialize(title, color, _items, id = Random.rand(1..1000))
+    @id = id
+    @title = title
+    @color = color
+    @items = []
```

Comment:
I am not really sure about this change of mine, but I remember that _we cannot put in the initializer_ the variables that connect the **Item** class with the **Author**, **Label**, **Source**, and **Genre** classes and vice versa. We need to put them in a custom setter method. In this case, I think the method is `add_items`.
So I think that you can delete `line 8` and initialize the variable inside the `add_items` method. But again, I am not really sure.